crates/metaslang/graph_builder/src/lib.rs

```diff
@@ -98,6 +98,8 @@ pub mod graph;
 mod parser;
 mod variables;
 
+pub mod stack_graph;
```

Comment:
I suggest renaming the `stack_graph` module to something else, to differentiate it from the imported external `stack_graphs` crate that is used here. I was a bit confused trying to go through this code/understand it because of this.
Maybe something like `builder`?
```suggestion
pub mod builder;
```

Reply:
Agree it's confusing. But as commented above, I'd like us to rename and restructure the crate. Otherwise, `builder` can be as confusing when it's inside `metaslang_graph_builder`.